core/impl/src/main/java/org/jboss/portletbridge/config/WebXmlProcessor.java

```diff
@@ -354,7 +354,10 @@ protected ContentHandler getNextHandler(String uri, String localName, Attributes
 
             @Override
             protected void endLastElement() {
-                errorPages.put(exceptionType.toString().trim(), location.toString().trim());
+                // PBR-575 - Just skip non exception-type error-pages
+                if(exceptionType.toString()!=null && exceptionType.toString().trim().length() > 0) {
```

Comment:
`trim().length() > 0` could be changed to `isEmpty()`, but it's OK to keep it like this.